### PR TITLE
Remove information and configuration sections in envs

### LINKED ISSUE #127: Remove information and configuration sections in envs
Instead we should have a single `envs` section that contains an array of environment variables. The old "information" envs should have a new key added: `information` (or similar) with boolean value.

This makes it easier to understand the image.yaml.

### PR DESCRIPTION
**Pull request by goldmann**
Both sections were merged into one. Currently the only dfference
between "information" and "confguration" environment variables
is lackoof the "value" field. In case there is no value field -
it will not be added to the generated Dockerfile.

Fixes #127

### CODE REVIEW

File: dogen/schema/kwalify_schema.yaml [hunk 1]
```diff
@@ -23,22 +23,14 @@ map:
...
-          - map:
-              name: {type: str, required: True}
-              example: {type: any}
-              description: {type: str}
+    type: seq
+    sequence:
+      - type: map
+        mapping:
+          name: {type: str, required: True}
+          value: {type: any}
```
Comment (dbecvarik) [inline dogen/schema/kwalify_schema.yaml]: why not type: str?

Comment (goldmann) [inline dogen/schema/kwalify_schema.yaml]: This would prevent us from having integer values in yaml.

Comment (dbecvarik) [inline dogen/schema/kwalify_schema.yaml]: but theyre string in the end in bash - so better than conversion, right?

Comment (goldmann) [inline dogen/schema/kwalify_schema.yaml]: Correct, but why we should force people to quote anything? Besides this we loose information that this specific value is an integer.

Comment (dbecvarik) [inline dogen/schema/kwalify_schema.yaml]: wil it work for stuff like 22.0 too?

Comment (goldmann) [inline dogen/schema/kwalify_schema.yaml]: Yes.

**Pull request merged**